Modules/Core/Common/include/itkLoggerThreadWrapper.h

```diff
@@ -58,12 +58,12 @@ class ITK_TEMPLATE_EXPORT LoggerThreadWrapper:public SimpleLoggerType
   using DelayType = unsigned int;
 
   /** Definition of types of operations for LoggerThreadWrapper. */
-  typedef enum {
-    SET_PRIORITY_LEVEL,
-    SET_LEVEL_FOR_FLUSHING,
-    ADD_LOG_OUTPUT,
-    WRITE
-    } OperationType;
+  enum OperationType : uint8_t {
```

Comment:
@mseng10 Why not enum class?

Reply:
When changed to enum class I get the build error below:
`Undefined symbols for architecture x86_64:
  "itk::LoggerThreadWrapper<SimpleLogger>::ADD_LOG_OUTPUT", referenced from:
      itk::LoggerThreadWrapper<SimpleLogger>::AddLogOutput(itk::LogOutput*) in itkLoggerThreadWrapperTest.cxx.o
  "itk::LoggerThreadWrapper<SimpleLogger>::SET_PRIORITY_LEVEL", referenced from:
      itk::LoggerThreadWrapper<SimpleLogger>::SetPriorityLevel(itk::LoggerBase::PriorityLevelType) in itkLoggerThreadWrapperTest.cxx.o
  "itk::LoggerThreadWrapper<SimpleLogger>::SET_LEVEL_FOR_FLUSHING", referenced from:
      itk::LoggerThreadWrapper<SimpleLogger>::SetLevelForFlushing(itk::LoggerBase::PriorityLevelType) in itkLoggerThreadWrapperTest.cxx.o
  "itk::LoggerThreadWrapper<SimpleLogger>::WRITE", referenced from:
      itk::LoggerThreadWrapper<SimpleLogger>::Write(itk::LoggerBase::PriorityLevelType, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&) in itkLoggerThreadWrapperTest.cxx.o
ld: symbol(s) not found for architecture x86_64`

Reply:
Resolved.